mgnify-pipelines-toolkit/1.0.2/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM mambaorg/micromamba:1.5.10-noble
```

Slightly smaller and more recent build if you use 
```suggestion
FROM mambaorg/micromamba:2.0.8-alpine3.19
```

---

ahh.. yeah, I'll update. I'm not a fan of the alpine based ones, they break in weird ways with bioinfo tools sometimes